backend/src/models/category.ts

```diff
@@ -0,0 +1,10 @@
+export type Category = {
+  category_id: number;
+  user_id: number;
+  category_name: string;
+  amount: number;
+  monthly_default: number | null;
+};
+
+export type CreateCategoryPayload = Omit<Category, 'category_id' | 'user_id'>;
```

Comment:
Used for when sending a post request to create a category, the payload has some attributes of category not all. Omit just takes the type and removes the fields that you specify